### DISCUSSION TITLE
How can I impersonate an address for testing purposes?

### DISCUSSION THREAD

**Discussion by joelamouche** [ConsenSys-archive/ganache#4079] [Q&A]
As advertised in the README.md, it should be possible to impersonate any account without private keys. 

How do I do that? 

Thanks in advance

**Comment by davidmurdoch** [comment]
You can use `evm_addAccount`, https://github.com/trufflesuite/ganache/blob/develop/src/chains/ethereum/ethereum/RPC-METHODS.md#evm_addaccount or the command line option `--wallet.unlockedAccounts=0x<address here>` (see `ganache --help`)
